packages/eslint-plugin/docs/rules/no-namespace.mdx

```diff
@@ -137,6 +137,9 @@ You might consider using [ESLint disable comments](https://eslint.org/docs/lates
 
 ## Further Reading
 
+{/* cspell:disable-next-line */}
```

Comment:
FYI I think you can configure cspell to ignore anchors using ignore patterns

Reply:
oh?? OH?? That would be so nice...

I can't find anything Googling, but if you have a reference I would _very very happily_ send a PR to enable it & remove any now-unnecessary disables/ignores.